src/ApiOperations/Request.php

```diff
@@ -58,24 +58,30 @@ protected static function validateParams($params = [], $requiredParams = [])
      * @return array
      * @throws \Xendit\Exceptions\ApiException
      */
-    protected static function _request($method,
+    protected static function _request(
+        $method,
         $url,
         $params = []
     ) {
         $headers = [];
+        $params = array_change_key_case($params);
```

Comment:
Applied `array_key_change_case` to __request() `$params`, what it does is it converts all keys in the array to be lower case. This just provides flexibility for other params headers checking.

So the user can either send `X-API-VERSION` or `x-api-version` and it would be added as a header.

The problem I see before is if I pass `x-idempotency-key` in lowercase, it wouldn't be added in the header.

Reply:
I think this is rather a backend concern, for the client let's just keep it as-is

Reply:
I don't think this is a good idea to apply to all params, let's just keep it as-is

Reply:
Copy, reverted.